### PR TITLE
fixed omission of channel group on json

### PR DESCRIPTION
**Pull request by polwex**
When scrying through the airlock, `diary` and `heap` channels were not showing the groups they belong to. That makes interop very hard. Please consider merging.

### CODE REVIEW

File: desk/lib/diary-json.hoon [hunk 1]
```diff
@@ -96,6 +96,7 @@
     |=  p=perm:d
     %-  pairs
     :~  writers/a/(turn ~(tap in writers.p) (lead %s))
+        group/(flag group.p)
```
Comment (tomholford) [inline desk/lib/diary-json.hoon:99]:
```suggestion
        flag/(flag group.p)
```

File: desk/lib/heap-json.hoon [hunk 2]
```diff
@@ -60,6 +60,7 @@
     |=  p=perm:h
     %-  pairs
     :~  writers/a/(turn ~(tap in writers.p) (lead %s))
+        group/(flag group.p)
```
Comment (tomholford) [inline desk/lib/heap-json.hoon:63]:
```suggestion
        flag/(flag group.p)
```

**Review by tomholford — CHANGES_REQUESTED** [review]
Thanks for the PR, this will be useful for other integrations. Requested a minor rename inline (`group` > `flag`). Also, please update the corresponding `DiaryPerm` and `HeapPerm` Typescript types in `ui/src/types/diary.ts` and `ui/src/types/heap.ts`. Otherwise LGTM

**Comment by arthyn** [comment]
@tomholford we probably should stay with `group` as the key to stay consistent with chat. Also makes it more clear what it's for.

**Comment by tomholford** [comment]
@arthyn I addressed the PR feedback from both of us so we can close this one out. PTAL at your convenience

**Comment by tomholford** [comment]
@jamesacklin Per discussion in standup, this PR does not need a state migration since it is not modifying any types. Good for merge upon approval

**Review by tomholford — APPROVED** [review]
LGTM

edit: @arthyn since I made some changes, PTAL at your convenience

**Pull request merged**